app/services/post_alerter.rb

```diff
@@ -657,6 +657,10 @@ def notify_pm_users(post, reply_to_user, notified)
         create_notification(user, Notification.types[:private_message], post, skip_send_email_to: emails_to_skip_send)
       when TopicUser.notification_levels[:tracking]
         notify_group_summary(user, post)
+      when TopicUser.notification_levels[:regular]
```

Comment:
Should we apply this quote/reply logic to `tracking` as well? IMO `notify_group_summary` isn't really strong enough for quote/reply notifications.

Reply:
Sounds like a good improvement!